package.json

```diff
@@ -11,13 +11,30 @@
     "emoji",
     "react"
   ],
-  "author": "Arron Hunt",
+  "author": "Amos Tan",
```

Comment:
Same thing here, shouldn't replace original author, we can add contributors if we want.

See https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors